microsoft-edge/devtools-guide-chromium/whats-new/2022/06/devtools-103.md

```diff
@@ -0,0 +1,158 @@
+In Microsoft Edge 103, when taking a heap snapshot in the **Memory** tool, there are two new node types:
+*  **Object shape** - Refers to the hidden classes and descriptor arrays that V8 (the JavaScript engine of Microsoft Edge) uses to understand and index the properties on objects. 
+*  **BigInt** - Used to represent and manipulate values that are too large to be represented by the **Number** node type.
+
+![BigInt and Object Shape memory node types.](devtools-103-images/memory-node-types.png)
+
+For history, see [Issue 12769: Improve categorization of objects in heap snapshots](https://bugs.chromium.org/p/v8/issues/detail?id=12769).
+
+See also:
+* [Constructor (group) entries in the heap profiler](../../../memory-problems/heap-snapshots.md#constructor-group-entries-in-the-heap-profiler) in _Record heap snapshots using the Memory tool_.
+* [HiddenClasses and DescriptorArrays](https://v8.dev/blog/fast-properties#hiddenclasses-and-descriptorarrays) - Fast properties in V8.
+* [BigInt object](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/BigInt).
+
+
+<!-- ====================================================================== -->
+## DevTools extension for Visual Studio Code: Right-click an HTML file to launch, and Application tool
+
+<!-- Title: DevTools for Visual Studio Code: Right-click to launch, and Application tool -->
+<!-- Subtitle: Use the context menu on any HTML document in Visual Studio Code to launch a new instance of the Microsoft Edge browser and DevTools. -->
+
+To make it easier to start a new embedded instance of Microsoft Edge from within Visual Studio Code, when you right-click an HTML file in the **Explorer**, there's now an **Open with Edge** submenu that has the commands **Open Browser** and **Open Browser with DevTools**:
+
+![Right-click an HTML file to open a browser instance and the 'Edge DevTools' tab.](devtools-103-images/edge-devtools-rightclick-html-file.png)
+
+Also, you can now also use the **Application** tool to gain insights into the storage and service worker information of your project, without leaving Visual Studio Code:
+
+![The Application tool in the 'Edge DevTools' tab within Visual Studio Code.](devtools-103-images/edge-devtools-application-tool.png)
+
+See also:
+* [Microsoft Edge DevTools extension for Visual Studio Code](../../../../visual-studio-code/microsoft-edge-devtools-extension.md)
+<!-- * [Microsoft Edge DevTools for Visual Studio Code](https://aka.ms/devtools-for-code) at Marketplace -->
+
+
+<!-- ====================================================================== -->
+## Bug fixes for Focus Mode UI
+
+<!-- Title: Bug fixes for Focus Mode UI -->
+<!-- Subtitle: Various bugs impacting DevTools controls and keyboard shortcuts were fixed in the new Focus Mode UI. -->
+
+In the new **Focus Mode** DevTools UI, various bugs affecting controls and keyboard shortcuts for DevTools have been resolved:
+*  Improved behavior of collapsible tabs in the **Elements** tool.
```

Comment:
Can we get more specific here? @hxlnt, is this mostly about the tabs in the Styles pane? Or something else in the Elements tool that's collapsible?

Reply:
Proposed fix:  Improved behavior of collapsible side-panel tabs such as Styles in the **Elements** tool